Mathlib/Geometry/Manifold/AnalyticManifold.lean

```diff
@@ -3,171 +3,9 @@ Copyright (c) 2023 Michael Lee. All rights reserved.
-      AnalyticOnNhd 𝕜 (I ∘ e.symm ∘ I.symm) (I '' e.target) := by
-  simp only [mem_analyticGroupoid, I.range_eq_univ, inter_univ, I.image_eq]
-  rw [IsOpen.analyticOn_iff_analyticOnNhd, IsOpen.analyticOn_iff_analyticOnNhd]
-  · exact I.continuous_symm.isOpen_preimage _ e.open_target
-  · exact I.continuous_symm.isOpen_preimage _ e.open_source
-
-/-- `analyticGroupoid` is closed under products -/
-theorem analyticGroupoid_prod {E A : Type} [NormedAddCommGroup E] [NormedSpace 𝕜 E]
-    [TopologicalSpace A] {F B : Type} [NormedAddCommGroup F] [NormedSpace 𝕜 F]
-    [TopologicalSpace B] {I : ModelWithCorners 𝕜 E A} {J : ModelWithCorners 𝕜 F B}
-    {f : PartialHomeomorph A A} {g : PartialHomeomorph B B}
-    (fa : f ∈ analyticGroupoid I) (ga : g ∈ analyticGroupoid J) :
-    f.prod g ∈ analyticGroupoid (I.prod J) := by
-  have pe : range (I.prod J) = (range I).prod (range J) := I.range_prod
-  simp only [mem_analyticGroupoid, Function.comp, image_subset_iff] at fa ga ⊢
-  exact ⟨AnalyticOn.prod
-      (fa.1.comp analyticOn_fst fun _ m ↦ ⟨m.1.1, (pe ▸ m.2).1⟩)
-      (ga.1.comp analyticOn_snd fun _ m ↦ ⟨m.1.2, (pe ▸ m.2).2⟩),
-    AnalyticOn.prod
-      (fa.2.comp analyticOn_fst fun _ m ↦ ⟨m.1.1, (pe ▸ m.2).1⟩)
-      (ga.2.comp analyticOn_snd fun _ m ↦ ⟨m.1.2, (pe ▸ m.2).2⟩)⟩
-
-end analyticGroupoid
-
-section AnalyticManifold
-
-/-- An analytic manifold w.r.t. a model `I : ModelWithCorners 𝕜 E H` is a charted space over `H`
-s.t. all extended chart conversion maps are analytic. -/
-class AnalyticManifold (I : ModelWithCorners 𝕜 E H) (M : Type*) [TopologicalSpace M]
-  [ChartedSpace H M] : Prop extends HasGroupoid M (analyticGroupoid I)
-
-/-- Normed spaces are analytic manifolds over themselves. -/
-instance AnalyticManifold.self : AnalyticManifold 𝓘(𝕜, E) E where
-
-/-- `M × N` is an analytic manifold if `M` and `N` are -/
-instance AnalyticManifold.prod {E A : Type} [NormedAddCommGroup E] [NormedSpace 𝕜 E]
-    [TopologicalSpace A] {F B : Type} [NormedAddCommGroup F] [NormedSpace 𝕜 F]
-    [TopologicalSpace B] {I : ModelWithCorners 𝕜 E A} {J : ModelWithCorners 𝕜 F B}
-    {M : Type} [TopologicalSpace M] [ChartedSpace A M] [m : AnalyticManifold I M]
-    {N : Type} [TopologicalSpace N] [ChartedSpace B N] [n : AnalyticManifold J N] :
-    AnalyticManifold (I.prod J) (M × N) where
-  compatible := by
-    intro f g ⟨f1, f2, hf1, hf2, fe⟩ ⟨g1, g2, hg1, hg2, ge⟩
-    rw [← fe, ← ge, PartialHomeomorph.prod_symm, PartialHomeomorph.prod_trans]
-    exact analyticGroupoid_prod (m.toHasGroupoid.compatible f2 g2)
-      (n.toHasGroupoid.compatible hf2 hg2)
-
-/-- Analytic manifolds are smooth manifolds. -/
-instance AnalyticManifold.isManifold [ChartedSpace H M]
-    [cm : AnalyticManifold I M] :
-    IsManifold I ∞ M where
-  compatible hf hg := ⟨(cm.compatible hf hg).1.contDiffOn I.uniqueDiffOn_preimage_source,
-    (cm.compatible hg hf).1.contDiffOn I.uniqueDiffOn_preimage_source⟩
-
-
-end AnalyticManifold
+deprecated_module (since "2025-04-13")
```

Comment:
Can you add a deprecation message, explaining that analytic manifolds are integrated into the smoothness hierarchy now --- so one should use `IsManifold I \omega M` instead of `AnalyticManifold I M`?